appveyor.yml

```diff
@@ -0,0 +1,22 @@
+environment:
+  matrix:
+    - nodejs_version: "4"
```

Comment:
let's just not support node 4 on windows. I don't think we'll be able to do this.